File: TestClient/src/TestLibrary/Client/ClientData.java
package TestLibrary.Client;

import org.simpleframework.xml.Attribute;
import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

/**
 * Created by Pit on 13.06.13.
 */
@Root(name = "Client")
public class ClientData
{
    @Attribute(name = "Request")
    public String _request = "SETRESULT";

    @Attribute(name = "Name")
    public String Name;

    @Attribute(name = "LastName")
    public String LastName;

    @Attribute(name = "Group")
    public String Group;

    @Element(name = "Report")
    public ClientReport Report = new ClientReport();
}

File: AllSharpProject/SimpleTestServerUDP/DataManagers/ClientData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Xml.Linq;
using TestLibrary;

namespace SimpleTestServerUDP.DataManagers
{


    public struct ClientData
    {


        /// <summary>
        /// состояние клиента
        /// </summary>
        public Status Status;

        /// <summary>
        /// текущий xml файл 
        /// данного клиента
        /// </summary>
        public XDocument CurrentXml;

        /// <summary>
        /// имя клиента
        /// </summary>
        public string Name;

        /// <summary>
        /// фамилия клиента
        /// </summary>
        public string LastName;

        /// <summary>
        /// группа клиента
        /// </summary>
        public string Group;

        /// <summary>
        /// Gets the Buffer.
        /// </summary>
        public byte[] Buffer;

        /// <summary>
        /// Result string from client
        /// </summary>
        public string Result;

        /// <summary>
        /// ответ клиента
        /// </summary>
        //public ClientAnswerMain ClientAnswerResult;

        /// <summary>
        /// результат теста
        /// </summary>
        //public Result ClientResult;


    }
}
